website/docs/r/vapp_org_network.html.markdown

```diff
@@ -0,0 +1,42 @@
+---
+layout: "vcd"
+page_title: "vCloudDirector: vcd_vapp_org_network"
+sidebar_current: "docs-vcd-resource-vapp-org-network"
+description: |-
+  Provides capability to attach an existing Org VDC Network to a vApp and toggle network features.
+---
+
+# vcd\_vapp\_org\_network
+
+ Provides capability to attach an existing Org VDC Network to a vApp and toggle network features.
+
+Supported in provider *v2.7+*
+
+## Example Usage
+
+```hcl
+resource "vcd_vapp_org_network" "vappOrgNet" {
+  org = "my-org" # Optional
+  vdc = "my-vdc" # Optional
+
+  vapp_name         = "my-vapp"
+ # Comment below line to create an isolated vApp network
```

Comment:
Let's add a newline to make the new parameter (and comment) stand-out.

Reply:
done